labs/fabric-java-token-sdk.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Lab Proposal Document
+[//]: # (parent: Labs)
```

Comment:
```suggestion
parent: Labs
```

Reply:
Hi, What are the missing commits that were not signed off? 
I only have one under my name and it was signed-off. Or I'm missing something?

Reply:
DCO on this PR is failing.

Reply:
Based on https://github.com/hyperledger-labs/hyperledger-labs.github.io/pull/253/checks?check_run_id=18539446068, it looks like it is somehow not liking the Signed-off-by message on the commit in this PR. Unfortunately, I cannot tell what it is. Maybe Joao Antunes vs. João Antunes. Not sure if the tilde is causing a problem.

Reply:
I have no idea what I'm doing wrong.
I'm using Intellij to commit. Is it impacting the DCO validation?